examples/illustrations/cell_site/src/cell_site-site.json

```diff
@@ -0,0 +1,43 @@
+[
+    {
+        "@id": "kb:dc9b8413-f681-4bc6-a66e-b70a7ecde4d4",
+        "@type": "drafting:CellSite",
+        "uco-core:hasFacet": [
+            {
+                "@type": "drafting:CellSiteFacet",
+                "drafting:cellSiteType": "GSM",
+                "drafting:cellSiteCountryCode": "228",
+                "drafting:cellSiteNetworkCode": "2",
+                "drafting:cellSiteLocationAreaCode": "22100",
+                "drafting:cellSiteIdentifier": "29220952"
+            },
+            {
+                "@type": "drafting:AntennaFacet",
+                "drafting:azimuth": {
+                    "@type": "xsd:decimal",
+                    "@value": "45"
+                },
+                "drafting:elevation": {
+                    "@type": "xsd:decimal",
+                    "@value": "5"
+                },
+                "drafting:skew": {
+                    "@type": "xsd:decimal",
+                    "@value": "10"
+                },
+                "drafting:horizontalBeamWidth": {
+                    "@type": "xsd:decimal",
+                    "@value": "110"
+                },
+                "drafting:signalStrength": {
+                    "@type": "xsd:decimal",
+                    "@value": "20"
+                },
+                "drafting:height": {
+                    "@type": "xsd:decimal",
+                    "@value": "25"
+                }
```

Comment:
This should probably become `drafting:antennaHeight`.  No sense allowing for ontologically mixing up a person's height and an antenna's height.
We should also include a unit in the `rdfs:comment` definition.

Reply:
Oh!  I apparently missed you'd already specified a unit in the documenting comment.  Sorry about that.

Reply:
`height` has become `antennaHeight`.